src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -1827,6 +1827,11 @@ public static string DescribeSet(string set)
                 return DescribeChar(set[SetStartIndex]);
             }
 
+            if (set == AnyClass)
+            {
+                return ".";
+            }
```

Comment:
This will mean that if someone writes `[\0-\uFFFF]`, it'll render as '.', which is actually '[^\n]` unless `RegexOptions.Singleline` is set.  This doesn't seem like an improvement.